libs/perun/components/src/lib/members-list/members-list.component.ts

```diff
@@ -170,6 +175,12 @@ export class MembersListComponent implements OnChanges, AfterViewInit {
     this.displayedColumns = this.displayedColumns.filter(x => !this.hideColumns.includes(x));
     this.dataSource = new MatTableDataSource<RichMember>(this.members);
     this.setDataSource();
+    this.disabledRouting = this.disableRouting;
```

Comment:
I am just curious. Is this line needed?

Reply:
disabledRouting is used to disable routing when clicking on a button located on a row

Reply:
I know the purpose of the variable. What Ido not understand is why do you assign this variable to itself.